<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rubrica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAggiungi">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>230</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Aggiungi</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnElimina">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>430</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Elimina</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+D</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputNome">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>60</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Nome e Cognome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTelefono">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>110</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Numero di telefono</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputEmail">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>160</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>811</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModifica">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>330</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Modifica</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+M</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Contatto da cercare</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
